### DISCUSSION TITLE
exportPathMap for /pages/api

### DISCUSSION THREAD

**Discussion by leon-do** [vercel/next.js#11375] [Help]
Next has a built in API route
https://nextjs.org/docs/api-routes/introduction

but it uses `/pages/api`

Is it possible to change the default path from `/api/*` to something else like `/myApi/*`?

I was thinking about adding it to exportPathMap
https://nextjs.org/docs/api-reference/next.config.js/exportPathMap

Any suggestions?

**Comment by timneutkens** [comment]
You can't use `next export` with API routes as API routes (as the name implies) is meant for building APIs and a full static export does not have server processing.

You can use http://zeit.co/ or host using `next start` to leverage API routes.

This still allows for static pages to exists when using getStaticProps: https://nextjs.org/blog/next-9-3#next-gen-static-site-generation-ssg-support
